### ISSUE TITLE
Push notifications 

### ISSUE THREAD

**Issue by Emmitts89** [jcaruso001/MessagePro#14]
Add push notifications!

**Comment by jcaruso001** [comment]
What was the name of the notification service you wanted to use?

**Comment by Emmitts89** [comment]
Pushwoosh is the name of the service. 

www.pushwoosh.com
Username: messagepro
Password: RaftWoren33!

**Comment by jcaruso001** [comment]
I added the code specified below: 

https://www.pushwoosh.com/programming-push-notification/android/native-android-sdk-integration/

However it does require some keys from the Google developer console that I'll need acess to. We also may have to wait till we deploy beta there to test it.

**Comment by Emmitts89** [comment]
Push notifications were not working.

**Comment by jcaruso001** [comment]
Did you read what i wrote above?

**Comment by Emmitts89** [comment]
Yes I did see the note, however due to the verbiage saying it may have to wait, I tested it anyways and it was marked verified so I verified that it was not working.

**Comment by Emmitts89** [comment]
I am not getting any notification for reminders that are set for flagged messages or recap--is this included in this?

**Comment by jcaruso001** [comment]
No this is not part of pushwoosh
